js/app.js

```diff
@@ -1 +1,121 @@
-console.log('Hello frontend');
+// Logic Reference: https://github.com/angle943/tic-tac-toe
+// have bugs when meet condition 2=5=8, 3=6=9, 4=5=6, 7=8=9
+// add if(one) etc. to check if it is empty ''
+
+// Variables
+
+msg = document.querySelector('#message h2')
+xScores = document.getElementById('xScores')
+oScores = document.querySelector('#oScores')
+tieScores = document.querySelector('#tieScores')
+let xWins = 0
+let oWins = 0
+let tie = 0
+let moves = 0
+let next = ''
+let gameIsGoing = true
+console.log(xScores, oScores);
+
+// Functions
+
+// display winner
+function display(player) {
```

Comment:
Naming variables, functions, booleans, objects, and arrays can be very difficult.  You did great on line 16 with gameIsGoing for a boolean.  For this function, a better name than "display" might be something like "displayWinner" or "handleWin."  Naming things well can take some time but makes code a lot more readable and easier to write (because the logic will be clearer)!